docs/cheaha/tutorial/pytorch_tensorflow.md

```diff
@@ -0,0 +1,132 @@
+
+1. Access the CLI following the steps [here.](../../workflow_solutions/using_anaconda.md#installing-anaconda-environments-using-cheahas-command-line-interface-cli)
+
+<!-- markdownlint-disable MD046 -->
+!!! note
+
+    When installing packages, modules and libraries into environments, remember to also install `ipykernel` using `conda install ipykernel`. This way your activated environment would appear in the list of kernels in your Jupyter Notebook. 
+
+<!-- markdownlint-enable MD046 -->
+
+For a correct installation of pytorch, we have to ensure some conditions are met. See partition [docs](../hardware.md#details) for a guide. One of such conditions, is to load CUDA toolkit using the below command in your environment setup form (see image below).
+
+```bash
+
+module load CUDA/11.8.0
+
+```
+
+![!load CUDA](images/module_load_cuda.png)
+
+<!-- markdownlint-disable MD046 -->
+!!! note
+
+    The cudatoolkit version may vary, as at the time of this tutorial, 11.8 is the version used. Running `nvidia-smi`, as in the image below, will show you the status, version and other information on GPUs in your created job session. The CUDA version is highlighted. The GPU CUDA Version available on Cheaha at the time of this tutorial is 12.3. Because the toolkit version used is lower than the Cheaha GPU version, it works. 
+
+<!-- markdownlint-enable MD046 -->
+
+![!nvidia-smi output](images/CudaVersion.png)
+
+When your job has been created and your environment created and activated from the cheaha CLI (see above [instructions](../../workflow_solutions/using_anaconda.md#create-an-environment)), run the below command.
+
+```bash
+
+conda install pytorch torchvision torchaudio cudatoolkit=11.8 -c pytorch -c nvidia
+
+```
+
+This commands will install a GPU compatible PyTorch version into your environment. To verify PyTorch is installed, and to see what version you have installed in your environment, use the below command.
+
+```bash
+
+conda list | grep "torch"
+
+```
+
+You should get an output like the below image.
+
+![!PyTorch Env Output](images/pytorchversion_output.png)
+
+The same process can be followed for installing another Deep Learning library Tensorflow (see instructions [below](../tutorial/pytorch_tensorflow.md#steps-to-install-tensorflow)) with some minute differences. You may decide to install the TensorFlow library into the same environment or create a new one. As a best practice, you may want to install these libraries in different environments.
+
+## Using PyTorch on Jupyter Notebook
+
+As an example we will be using a sample Jupyter Notebook with just a simple torch function to test if a GPU will be utilized with PyTorch functions. Run the command in a cell, and if your output is `True`, then you have your GPU setup to support PyTorch functions.
+
+```bash
```

Comment:
Let's format this like Python code. Also this isn't bash so put `python` instead of `bash` at the opening fence.

```python

import torch

print(torch.cuda.is_available())
x = torch.cuda.current_device()
print(torch.get_device_name(x))

```